crates/rlp/Cargo.toml

```diff
@@ -18,6 +18,9 @@ ethereum-types = { version = "0.14", features = ["codec"], optional = true }
 revm-primitives = { workspace = true, features = ["serde"] }
 reth-rlp-derive = {  path = "./rlp-derive", optional = true }
 
+# for eip-4844
+c-kzg = { git = "https://github.com/ethereum/c-kzg-4844", optional = true }
```

Comment:
@DaniPopes wdyt here, how would this affect migration to alloy-rlp?

Reply:
Looks like it's just wrappers for byte arrays so it should be fine, but in general i don't think it's the rlp's crate job to provide implementations for external types

Reply:
the only other feasible solution would be to use wrapper types in reth which I think is messier

we could try to add it to the c-kzg repo itself eventually